.github/workflows/py-ingestion-core-publish.yml

```diff
@@ -24,6 +24,9 @@ on:
       - 'catalog-rest-service/src/main/resources/json/schema/**'
       - 'ingestion-core/src/metadata/_version.py'
 
+concurrency: 
+  group: ${{ github.head_ref }}
```

Comment:
`github.head_ref` is only defined on Pull Requests. Let's change this to a more common variable that will be available on all events and is unique